### PR TITLE
Gluttony morph cant ventcrawl

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request

Removes the ability to ventcrawl for morphs made from lavaland gluttony ruin.

## Why It's Good For The Game

Self made antags can't destroy the round as easily.

## Changelog
:cl:
balance: Gluttony Morph can no longer ventcrawl
/:cl:

### CODE REVIEW

File: code/modules/antagonists/morph/morph.dm [hunk 1]
```diff
@@ -54,6 +54,9 @@
 
 	mobchatspan = "blob"
 
+/mob/living/simple_animal/hostile/morph/discount
+	ventcrawler = null
```
Comment (Archanial) [inline code/modules/antagonists/morph/morph.dm]:
```suggestion
	ventcrawler = FALSE
```

**Comment by Nosnek199** [comment]
Gluttony morphs are antagonists.

**Review by PowerfulBacon — APPROVED** [review]
Code is fine, idea is bad

**Comment by zeskorion** [comment]
these are supposed to ventcrawl as their primary ability. Morphs are squishy, not that dangerous, and rely upon surprise

**Comment by utilbird** [comment]
ditto.

**Comment by Glcbitj** [comment]
"Bro, what if we removed one of the secondary antags two abilities lmao"

**Pull request closed without merging**